src/lib.rs

```diff
@@ -300,3 +300,9 @@ impl From<bool> for Toggle {
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
 pub struct TryFromIntError;
+
+fn stripped_type_name<T>() -> &'static str {
+    let s = core::any::type_name::<T>();
+    let p = s.split("::");
+    p.last().unwrap()
```

Comment:
To safe one unwrap call, this could be done, I guess:

```suggestion
    p.last().unwrap_or(s)
```